server/container/guice/guice-common/src/test/java/org/apache/james/AbstractJamesServerTest.java

```diff
@@ -61,9 +62,28 @@ public void setup() throws Exception {
 
     protected abstract GuiceJamesServer<Id> createJamesServer();
 
+    protected abstract void clean();
```

Comment:
I'm not sure, this method is dependent of the implementation as only the implementation knows how to clean its DB.
The Abstract class manages the lifecycle of the tests, so it should call the clean method.